sdk/core/src/error/mod.rs

```diff
@@ -257,6 +257,12 @@ impl From<url::ParseError> for Error {
     }
 }
 
+impl From<time::error::Parse> for Error {
```

Comment:
I'm unsure if we should have this. It might be nice to require that users add a context when this error occurs. Otherwise the error message might be a bit obtuse.

Reply:
I've removed `From<time::error::Parse>`  as requested.